SoundGrid.py

```diff
@@ -0,0 +1,184 @@
+"""Sound Grid
+	SoftDes Spring 2016 MP4: Interactive Programming
+	Gaby Clarke and Kristyn Walker"""
+
```

Comment:
Adding a short summary of what the file contains/does in this header/docstring is often helpful for others to quickly figure out what to expect/what your code does.